rollup/internal/orm/chunk.go

```diff
@@ -254,6 +256,52 @@ func (o *Chunk) InsertChunk(ctx context.Context, chunk *encoding.Chunk, codecVer
 	return &newChunk, nil
 }
 
+func (o *Chunk) InsertChunkRaw(ctx context.Context, index uint64, codecVersion encoding.CodecVersion, chunk *encoding.DAChunkRawTx, totalL1MessagePoppedBefore uint64) (*Chunk, error) {
```

Comment:
ditto

Reply:
done in 6ef477578a69c107d611a686eea7d7f486582e6e